packages/extension/src/view/popup/stateProviders/syncCookieStore/index.tsx

```diff
@@ -16,7 +16,7 @@
 /**
  * External dependencies.
  */
-import { useContextSelector, createContext } from 'use-context-selector';
```

Comment:
I am thinking, instead of importing `createContext` and `useContextSelector` from two different places, shall we import `createContext` from `@ps-analysis-tool/common` only ( export `createContext` from common package )